online-book-store/src/test/resources/application.yml

```diff
@@ -1,13 +0,0 @@
-spring:
```

Comment:
The build failed because of the annotation '@SpringBootTest' in the class OnlineBookStoreApplicationTests, we should remove it.